store/models.py

```diff
@@ -173,10 +174,14 @@ def add_line_from_data(self, orderlines_data):
                     option: BaseProduct = BaseProduct.objects.get(
                         id=option_id_quantity['id'])
                     quantity = option_id_quantity['quantity']
+                    metadata = None
+                    if 'metadata' in option_id_quantity:
+                        metadata = option_id_quantity['metadata']
```

Comment:
`option_ids_quntity` could be rename `options` or something better ?
It seems like this naming loose its purpose with the time and the refactors.